consoles/ssh_screen.pm

```diff
@@ -25,9 +25,8 @@ has ssh_channel    => undef;
 
 use constant TYPE_STRING_TIMEOUT => ONE_MINUTE;
 
-sub new {
-    my $class = shift;
-    my $self  = bless @_ ? @_ > 1 ? {@_} : {%{$_[0]}} : {}, ref $class || $class;
+sub new ($class) {
+    my $self = bless @_ ? @_ > 1 ? {@_} : {%{$_[0]}} : {}, ref $class || $class;
```

Comment:
I don't think this is going to work. The function will only accept one argument and is `@_` really implicitly shifted?

Reply:
Right. Changed that to a second argument `@args`